plugins/ipam/ipstore/ipstore.go

```diff
@@ -159,6 +167,53 @@ func (manager *IPManager) Release(ip string) error {
 	return nil
 }
 
+// ReleaseByValue release the key-value pair by value
+func (manager *IPManager) ReleaseByValue(value string) (string, error) {
+	// TODO improve this part by implement listing all the kv pairs
+	// libkv library only provide list all key-value pairs with prefix of key, and
+	// if the result is empty it will return store.ErrKeyNotFound error
+	kvPairs, err := manager.client.List("1")
```

Comment:
What is the meaning of this string (`"1"`)? Can you const this and add a comment explaining the same?